docs/synapse/devopsguide.rst

```diff
@@ -182,6 +182,38 @@ include a :ref:`datamigration` while it comes back online::
 
     Once a Synapse service update has been deployed, you may **NOT** revert to a previous version!
 
+.. _devops-task-storm-pool:
+
+Configure a Storm Query Pool
+----------------------------
```

Comment:
This section should be moved after the data migration/model flag day subsections
![image](https://github.com/vertexproject/synapse/assets/3729414/df2e7447-c19a-4707-a414-d8a0913424c4)

Reply:
lol, nice catch, thanks!